main/lsp/AutocorrectFlusher.cc

```diff
@@ -0,0 +1,19 @@
+#include "main/lsp/LSPTypechecker.h"
+
+using namespace std;
+namespace sorbet::realmain::lsp {
+
+void AutocorrectFlusher::flushErrors(spdlog::logger &logger, vector<unique_ptr<core::ErrorQueueMessage>> errors,
```

Comment:
WDYT about renaming this to `ErrorCollector`? It's only used for autocorrect, but it's not actually flushing autocorrects. (It's not actually flushing anything.)